src/main/java/net/snowflake/client/jdbc/SnowflakeUtil.java

```diff
@@ -522,6 +522,42 @@ public static String systemGetEnv(String env) {
     return null;
   }
 
+  /** System.setEnv function. Can be used for unit tests. */
```

Comment:
Added functions for unit testing for setting and unsetting system environment variables.